pkg/k8s/watchers/node.go

```diff
@@ -37,6 +46,18 @@ func (k *K8sWatcher) RegisterNodeSubscriber(s subscriber.Node) {
 	k.NodeChain.Register(s)
 }
 
+func nodeEventsAreEqual(oldNode, newNode *v1.Node) bool {
+	if !comparator.MapStringEquals(oldNode.GetLabels(), newNode.GetLabels()) {
+		return false
+	}
+
+	if !reflect.DeepEqual(oldNode.Status.Addresses, newNode.Status.Addresses) {
+		return false
+	}
+
```

Comment:
If the title says we won't do anything for addresses then we can drop this?

Reply:
Correct, this is not needed now, I'll drop it :+1: